install.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh
+
+if [[ -n $1 ]] && [[ $1 == '--scripts' ]]; then
+  echo '💡  Installing scripts…'
+
+  PATH_LINE='export PATH=$PATH:$HOME/.git-semantic-commits:$PATH'
+  BASHRC=~/.bashrc
+  ZSHRC=~/.zshrc
+
+  if [ -f $BASHRC ]; then
+    if ! grep -Fxq "$PATH_LINE" $BASHRC
+    then
+      echo
+      echo "📝  Adding path to $BASHRC"
+      echo $PATH_LINE >> $BASHRC
+      source $BASHRC
+    fi
+  fi
+
+  if [ -f $ZSHRC ]; then
+    if ! grep -Fxq "$PATH_LINE" $ZSHRC
+    then
+      echo
+      echo "📝  Adding path to $ZSHRC"
+      echo $PATH_LINE >> $ZSHRC
+      source $ZSHRC
+    fi
+  fi
+else
+  echo '💡  Installing git aliases…'
+
+  # git-extras/chore compatibility https://goo.gl/oZUN3V
+  if ! git config --global --get-all alias.ch &>/dev/null; then
+    git config --global alias.ch '!f() { [[ -z "$GIT_PREFIX" ]] || cd "$GIT_PREFIX" && if [[ -z $1 ]]; then git commit -m "chore: " -e; else git commit -m "chore: $1"; fi }; f'
+  fi
+
+  if ! git config --global --get-all alias.chore &>/dev/null; then
+    git config --global alias.chore '!f() { [[ -z "$GIT_PREFIX" ]] || cd "$GIT_PREFIX" && if [[ -z $1 ]]; then git commit -m "chore: " -e; else git commit -m "chore: $1"; fi }; f'
+  fi
```

Comment:
Fair enough. Now it detects if there is `git-extras` an only then provides aliases for `chore` and `refactor` commands (➕ the same for respective files).